### PR TITLE
Updates to README ahead of `v12`.

### PR DESCRIPTION
**Pull request by adamrusted**
This PR serves two functions:
- Split extensions to 3rd Party tooling out into their own category.
- Remove libraries that have not kept up to date with the latest releases.

## Removed Extensions

> [!IMPORTANT]\
> Removal of extensions from the list just means we don't 'endorse' them, given they are not keeping up with our legal obligations to remove icons when requested by brands. They can be added back in as/when they become compliant again. Maintainers of these extensions have had notice since v10 of this shift.

- Angular
- Drupal
- Flutter
- Hexo
- Home Assistant
- Java
- Jetpack Compose
- Leptos
- Python
- Solid
- Vue
- WordPress

### CODE REVIEW

**Review by uncenter — APPROVED** [review]

**Review by LitoMore — APPROVED** [review]

**Comment by adamrusted** [comment]
@uncenter @LitoMore - have just amended to include changes to the tests.

File: sdk.mjs [hunk 1]
```diff
@@ -196,7 +196,42 @@ export const getThirdPartyExtensions = async (
...
+  readmePath = path.join(
+    getDirnameFromImportMeta(import.meta.url),
+    'README.md',
+  ),
+) =>
+  normalizeNewlines(await fs.readFile(readmePath, 'utf8'))
+    .split(
+      '## Third-Party Libraries\n\nThe below are known third-party libraries for use in your own projects. We only keep items in the list that are at least up to date with our previous major version.\n\n',
+    )[1]
     .split('\n\n', 1)[0]
```
Comment (mondeja) [inline sdk.mjs:235]:
```suggestion
    .split('## Third-Party Libraries')[1]
    .split('\n\n')[2]
```

File: sdk.mjs [hunk 2]
```diff
@@ -196,7 +196,42 @@ export const getThirdPartyExtensions = async (
   ),
 ) =>
   normalizeNewlines(await fs.readFile(readmePath, 'utf8'))
-    .split('## Third-Party Extensions\n\n')[1]
+    .split(
+      '## Third-Party Extensions\n\nThe below are known extensions to third-party tools.\n\n',
+    )[1]
+    .split('\n\n', 1)[0]
```
Comment (mondeja) [inline sdk.mjs:202]:
```suggestion
    .split('## Third-Party Extensions')[1]
    .split('\n\n')[2]
```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Review by mondeja — APPROVED** [review]
LGTM

**Pull request merged**